distribution/lib/Standard/Database/0.0.0-dev/src/DB_Table.enso

```diff
@@ -90,6 +89,14 @@ type DB_Table
        - context: The context associated with this table.
     private Value internal_name:Text connection:(Connection | Any) (internal_columns:(Vector Internal_Column)) context:Context
 
+    ## The internal constructor used to construct a DB_Table instance.
+
+       It can perform some additional operations, like refining the type,
+       so it should always be preferred over calling `DB_Table.Value` directly.
```

Comment:
How would I know this without reading this comment? Guess we need some sort of convention for module private constructors similar to what we have for module priavte methods

Reply:
Good point, I will add a comment in `Value` constructor that it should not be used directly.